src/proposals/20241114/SparkEthereum_20241114.sol

```diff
@@ -0,0 +1,196 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { IERC20 }   from 'forge-std/interfaces/IERC20.sol';
+import { IERC4626 } from 'forge-std/interfaces/IERC4626.sol';
+
+import { Ethereum, SparkPayloadEthereum, IEngine, EngineFlags } from 'src/SparkPayloadEthereum.sol';
+
+import { ICapAutomator } from "lib/sparklend-cap-automator/src/interfaces/ICapAutomator.sol";
+
+import { IMetaMorpho, MarketParams } from 'lib/metamorpho/src/interfaces/IMetaMorpho.sol';
+
+import { Base } from 'spark-address-registry/Base.sol';
+
+import { CCTPForwarder }     from 'xchain-helpers/forwarders/CCTPForwarder.sol';
+import { OptimismForwarder } from 'xchain-helpers/forwarders/OptimismForwarder.sol';
+
+import { AllocatorBuffer } from 'dss-allocator/src/AllocatorBuffer.sol';
+import { AllocatorVault }  from 'dss-allocator/src/AllocatorVault.sol';
+
+import {
+    ControllerInstance,
+    MainnetControllerInit,
+    RateLimitData,
+    MintRecipient
+} from 'lib/spark-alm-controller/deploy/ControllerInit.sol';
+
+interface ITokenBridge {
+    function bridgeERC20To(
+        address _localToken,
+        address _remoteToken,
+        address _to,
+        uint256 _amount,
+        uint32 _minGasLimit,
+        bytes calldata _extraData
+    ) external;
+}
+
+/**
+ * @title  Nov 14, 2024 Spark Ethereum Proposal
+ * @notice Activate Spark Liquidity Layer, Morpho Supply Caps, WBTC/WETH/WSTETH Market Adjustments
+ * @author Phoenix Labs
+ * Forum:  https://forum.sky.money/t/14-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25466
+ * Vote:   https://vote.makerdao.com/polling/Qmf955yA
+ *         https://vote.makerdao.com/polling/QmXsXzot
+ *         https://vote.makerdao.com/polling/QmYqM8Yf
+ *         https://vote.makerdao.com/polling/QmNTKFqG
+ *         https://vote.makerdao.com/polling/Qmbohkr5
+ *         https://vote.makerdao.com/polling/QmQizL1F
+ */
+contract SparkEthereum_20241114 is SparkPayloadEthereum {
+
+    address internal constant PT_26DEC2024_PRICE_FEED = 0x81E5E28F33D314e9211885d6f0F4080E755e4595;
+    address internal constant PT_SUSDE_26DEC2024      = 0xEe9085fC268F6727d5D4293dBABccF901ffDCC29;
+
+    address internal constant PT_27MAR2025_PRICE_FEED = 0x38d130cEe60CDa080A3b3aC94C79c34B6Fc919A7;
+    address internal constant PT_SUSDE_27MAR2025      = 0xE00bd3Df25fb187d6ABBB620b3dfd19839947b81;
+
+    address internal constant WETH_IRM = 0xf4268AeC16d13446381F8a2c9bB05239323756ca;
+
+    address internal constant FREEZER = 0x90D8c80C028B4C09C0d8dcAab9bbB057F0513431;  // Gov. facilitator multisig
+    address internal constant RELAYER = 0x8a25A24EDE9482C4Fc0738F99611BE58F1c839AB;
+
+    uint256 internal constant USDS_MINT_AMOUNT     = 9_000_000e18;
+    uint256 internal constant SUSDS_DEPOSIT_AMOUNT = 8_000_000e18;
+    uint256 internal constant USDS_BRIDGE_AMOUNT   = 1_000_000e18;
```

Comment:
Where do these values come from? Didn't find it in the forum post

Reply:
Good catch. It's been added more explicitly: https://forum.sky.money/t/14-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25466#p-99720-mainnetbase-activate-spark-liquidity-layer-1